COMPILATION.md

```diff
@@ -6,32 +6,59 @@ If you want specific instructions for some distributions, check the [wiki](https
 
 Keep in mind using the pre-built packages when available.
 
-1. Ensure your system satisfies build and runtime dependencies for:
+## Compilation on Linux
+
+### Ensure your system satisfies build and runtime dependencies for:
```

Comment:
Should this be build time dependencies only?

Reply:
Does that mean we should also write `cppcheck` / `shellcheck` etc in this file?
The purpose of this `COMPILATION.md` file was not to perform checks or tests, but only to generate an executable file, so I considered it to be the minimum necessary.
Would it be better to add more detailed dependencies?